arrow-core/src/main/kotlin/arrow/core/extensions/ior.kt

```diff
@@ -72,19 +70,16 @@ interface IorSemigroup<L, R> : Semigroup<Ior<L, R>> {
     }
 }
 
-@extension
 @undocumented
 interface IorFunctor<L> : Functor<IorPartialOf<L>> {
```

Comment:
```
@Deprecated(
  message = "Functor typeclass is deprecated and will be removed in 0.13.0. Use concrete methods on Const",
  level = DeprecationLevel.WARNING
)
```

Typeclasses that are going to be removed/deprecated should be annotated with the above deprecation